vjailbreak-ui/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Logs
```

Comment:
In the interest of saving on typing and eliminating redundancy from the name, I would prefer if we could have this simply under a directory named `ui`.

Reply:
Resolved